partner_vat_unique/README.rst

```diff
@@ -39,13 +39,22 @@ Installation
 Will not check previous VAT duplicates, so it is recomended make sure there
 isn't any duplicated VAT before installation.
 
+Just verify the main contacts, as VAT is a commercial, field that is basically copied to the underlying contacts.
+
+Known issues / Roadmap
+======================
+
+* Do not allow the creation of duplicate VATs, you are currently notified but would let you create it.
```

Comment:
This is not what we have talked. These are the problems with the standard one. This module prevents effectively this. The only know issues is when partners are shared between several companies.